classes/class-wpcom-liveblog-entry-embed-sdks.php

```diff
@@ -39,7 +39,7 @@ public static function enqueue() {
 		}
 
 		foreach ( self::$sdks as $name => $url ) {
-			wp_enqueue_script( $name, esc_url( $url ), array(), WPCOM_Liveblog::VERSION, false );
+			wp_enqueue_script( $name, esc_url( $url ), array(), null, false );
```

Comment:
Can we leave it as-is, please? This should go out with a new release of Liveblog.

Reply:
problem is if version will add in reddit js file like https://embed.reddit.com/widgets.js?ver=123  it will not work 
then again it will generate 404 error , that's why i have removed or we can do ,we can add a conditional statement for this only in case of reddit version will be null and for other's version will add.

Reply:
@rebeccahum 👍 Please check now